### PR TITLE
Move to latest versions of SDK

### PR DESCRIPTION
**Pull request by chrimc62**
This updates to the latest SDK packages including bug fixes to LG and LU.
For LG, allow # in multi-line.
For LU, set line numbers for entities.

### CODE REVIEW

File: experimental/generation/generator/package.json [hunk 1]
```diff
@@ -5,15 +5,17 @@
   "author": "Microsoft",
   "bugs": "https://github.com/Microsoft/BotBuilder-Samples/issues",
   "dependencies": {
-    "@microsoft/bf-cli-command": "^4.8.0",
+    "@microsoft/bf-cli-command": "^4.9.0-RC6",
+    "@microsoft/bf-lu": "^4.10.0-preview.130225",
```
Comment (tomlm) [inline experimental/generation/generator/package.json:9]: >"^4.10.0-preview.130225", [](start = 24, length = 25)

this doesn't seem right.

File: experimental/generation/generator/package.json [hunk 2]
```diff
@@ -5,15 +5,17 @@
   "author": "Microsoft",
   "bugs": "https://github.com/Microsoft/BotBuilder-Samples/issues",
   "dependencies": {
-    "@microsoft/bf-cli-command": "^4.8.0",
+    "@microsoft/bf-cli-command": "^4.9.0-RC6",
```
Comment (tomlm) [inline experimental/generation/generator/package.json:8]: >", [](start = 44, length = 2)

this is no rc42?

File: experimental/generation/generator/templates/booleanEntity.en-us.lg.lg [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 # template
 - ```
-# booleanEntity(value)
+${'#'} booleanEntity(value)
```
Comment (tomlm) [inline experimental/generation/generator/templates/booleanEntity.en-us.lg.lg:8]: >${'#' [](start = 0, length = 5)

what the wha?

File: experimental/generation/generator/templates/enumArrayProperty.lg.lg [hunk 4]
```diff
@@ -15,11 +15,11 @@ ${ValueTemplate()}
...
 - ```
-# Ask${property}
-- \${askEnumArray('${property}')\}```
+${'#'} Ask${property}
+- \${askEnumArray('${property}')}```
 
 # ValueTemplate
 - ```
-# ${property}(values)
+${'#'} ${property}(values)
```
Comment (tomlm) [inline experimental/generation/generator/templates/enumArrayProperty.lg.lg:23]: >${'#'}  [](start = 0, length = 7)

can you \# ?

File: experimental/generation/runbot/TestBotHttpAdapter.cs [hunk 5]
```diff
@@ -22,11 +22,9 @@ public class TestBotHttpAdapter : BotFrameworkHttpAdapter
             : base(configuration, credentialProvider)
         {
             this.UseStorage(storage);
-            this.UseState(userState, conversationState);
+            this.UseBotState(userState, conversationState);
             this.UseDebugger(configuration.GetValue("debugport", 4712), logger: logger);
 
-            HostContext.Current.Set<IConfiguration>(configuration);
```
Comment (tomlm) [inline experimental/generation/runbot/TestBotHttpAdapter.cs:28]: >HostContext.Current.Set<IConfiguration>(configuration); [](start = 12, length = 55)

If you don't set this luis settings won't work. 
    this.Use(new RegisterClassMiddleware<IConfiguration>(configuration));

File: experimental/generation/generator/package.json (hunk 1 above)
Comment (chrimc62) [reply experimental/generation/generator/package.json:9]: That is correct.  I have to pick up bug fixes.

---
In reply to: [423897684](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423897684) [](ancestors = 423897684)

File: experimental/generation/generator/templates/booleanEntity.en-us.lg.lg (hunk 3 above)
Comment (chrimc62) [reply experimental/generation/generator/templates/booleanEntity.en-us.lg.lg:8]: Bug in LG that they are working on fixing and this is the work-around until they do.

---
In reply to: [423898362](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423898362) [](ancestors = 423898362)

File: experimental/generation/generator/templates/enumArrayProperty.lg.lg (hunk 4 above)
Comment (chrimc62) [reply experimental/generation/generator/templates/enumArrayProperty.lg.lg:23]: No. That then passes through as \#

---
In reply to: [423898807](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423898807) [](ancestors = 423898807)

File: experimental/generation/runbot/TestBotHttpAdapter.cs (hunk 5 above)
Comment (chrimc62) [reply experimental/generation/runbot/TestBotHttpAdapter.cs:28]: That is a bug I'm tracking down, so I will make that change.  
The issue was HostContext was not found.

---
In reply to: [423899800](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423899800) [](ancestors = 423899800)

**Pull request merged**